### PR TITLE
being more explicit for zone limitation

### PR DESCRIPTION
**Pull request by l0k0ms**
This is an update of: https://github.com/DataDog/documentation/pull/1638

Preview: https://d2cx6t4ssmwdrv.cloudfront.net/gus/dyn-update/integrations/dyn/

### CODE REVIEW

File: content/integrations/dyn.md [hunk 1]
```diff
@@ -34,14 +33,15 @@ Once you have created a Datadog Read only user:
 4. Configure your [Dyn integration](https://app.datadoghq.com/account/settings#integrations/dyn) inside your Datadog application.
 {{< img src="integrations/dyn/dyn_integration.png" alt="Dyn Integration" style="width:75%;" >}}
 
-5. Select your zones you want to monitor:
-{{< img src="integrations/dyn/dyn_zone.png" alt="Dyn zone" style="width:50%;" >}}
-
+5. Select your zones you wish to monitor:<br>
```
Comment (irabinovitch) [inline content/integrations/dyn.md]: Same feedback as last time "Select the zones you wish to monitor"

File: content/integrations/dyn.md [hunk 2]
```diff
@@ -34,14 +33,15 @@ Once you have created a Datadog Read only user:
...
+{{< img src="integrations/dyn/dyn_zone.png" alt="Dyn zone" style="width:75%;" >}}
+The “Zone limitation” applies only to “Zone notes”.
+**Dyn “QPS” metrics are by default collected for all zones**, Events and `dyn.changes` metric are not.
```
Comment (irabinovitch) [inline content/integrations/dyn.md]: Dyn “QPS” metrics are collected by default for all zones. , Events and `dyn.changes` metric are not.

Comment (irabinovitch) [inline content/integrations/dyn.md]: What does one need to do to collect Events and `dyn.changes` metric?

File: content/integrations/dyn.md [hunk 3]
```diff
@@ -34,14 +33,15 @@ Once you have created a Datadog Read only user:
...
 <div class="alert alert-info">
 IP ACLs must be disabled for the Dyn integration.
```
Comment (irabinovitch) [inline content/integrations/dyn.md]: Is this still true @yannmh ? I believe we can now provide a known IP block for crawlers. Can you confirm?

File: content/integrations/dyn.md (hunk 1 above)
Comment (l0k0ms) [reply content/integrations/dyn.md]: 👍

File: content/integrations/dyn.md (hunk 2 above)
Comment (l0k0ms) [reply content/integrations/dyn.md]: 👍

Comment (l0k0ms) [reply content/integrations/dyn.md]: I rephrased the 5th step.

**Pull request merged**